Design changing and random color event add

bluayer opened the pull request:
For more better design.

bluayer committed:
Design changing and random color event add

makerdark98 commented:
I agree to add `color` property to design
But I don't know how to display older `events` that don't have `color` property.
Please add exception handling in this situation

bluayer committed:
Patching for Random color in events

My last commit isn't considered the older data.
Changing database schema is not good for everyone as you know.
So i'm just using js in front-end for get random color and just use it in front-end.
It can be good solution for random color that not modified db.

bluayer commented:
I understand what you said.
I also agree that modifying db is so dangerous.
So i just using js in front-end for random coloring about events.
"Patching for Random color in events" commit is commit about that.
